core/src/executor/evaluate/function.rs

```diff
@@ -538,6 +539,19 @@ pub fn is_empty<'a>(expr: Evaluated<'_>) -> Result<Evaluated<'a>> {
     Ok(Evaluated::from(Value::Bool(length == 0)))
 }
 
+pub fn values<'a>(expr: Evaluated<'_>) -> Result<Evaluated<'a>> {
+    let expr: Value = expr.try_into()?;
+    match expr {
+        Value::Map(m) => Ok(Evaluated::from(Value::List(
+            m.into_iter()
+                .sorted_by(|(k1, _), (k2, _)| k1.cmp(k2))
```

Comment:
Finally, SORT function (#1300)  was merged so that we can remove internal sort here.
Then, let's wrap test cases with `SORT()`

```rs
sql: r#"SELECT SORT(VALUES(data)) as result FROM USER WHERE id=1"#,
```

Reply:
Thanks for reviews.
I updated values function to remove sorting logic and added `SORT()` function into test codes. But if the list consists of different types, it can not be ordered correctly every time.
So I want to handle this in another issue #1324.
Could you please review this PR again?